app/styles/_breakpoints.scss

```diff
@@ -0,0 +1,16 @@
+// Breakpoint management
+// http://www.sitepoint.com/managing-responsive-breakpoints-sass/
+$breakpoints: (
+  small: ('max-width: 520px'),
+  big: ('min-width: 521px')
+  );
+@mixin respond-to($breakpoint) {
+  @if map-has-key($breakpoints, $breakpoint) {
+    @media (#{map-get($breakpoints, $breakpoint)}) {
+      @content;
+    }
+  } @else {
+    @warn 'Unfortunately, no value could be retrieved from `#{$breakpoint}`. '
+    + 'Please make sure it is defined in `$breakpoints` map.';
+  }
+}
```

Comment:
I like the organization of breakpoints, but does the `@mixin` make more sense here, or in _./app/styles/_mixins.scss_?

I guess similarly, should we leave the breakpoints here in this file, or just in the _./app/styles/_variables.scss_ file?